src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs

```diff
@@ -44,7 +44,8 @@ public MutationTestInput Initialize(StrykerOptions options)
             var projectInfo = _inputFileResolver.ResolveInput(options);
 
             // initial build
-            _initialBuildProcess.InitialBuild(projectInfo.FullFramework, options.BasePath, options.SolutionPath, Path.GetFileName(projectInfo.TestProjectAnalyzerResult.ProjectFilePath));
+            _initialBuildProcess.InitialBuild(projectInfo.FullFramework, Path.GetDirectoryName(projectInfo.TestProjectAnalyzerResult.ProjectFilePath), 
+                options.SolutionPath, Path.GetFileName(projectInfo.TestProjectAnalyzerResult.ProjectFilePath));
```

Comment:
Same here but for the filename. I see we implemented it like this ourselves before, now that I notice I disagree with our implementation. Could you fix it here as well?